ruby/alias_manager.rb

```diff
@@ -0,0 +1,77 @@
+=begin
+ 	 
+-Create a method that swaps their first and last name
+-Create a method that changes all the vowels to the next vowel in 'aeiou'
+	-Create a loop that goes through all the letters of the name
+	-When the letter matches a letter in vowels list, 
+	change it to the next vowel in the vowels list
+	-Until all the letters are swapped, do the loop
+	-Save the newly swapped name in a variable
+-Create a method that changes all the rest of consonants to the next
+	-Iterate through all the letters and change the letter to the next one
+	in the consonants list.
+-Get user input for name and give the fake name as output until they say quit.
+-Print all the data user entered
+=end
+
+#Checking Felicia Torres 
+#real_name = "Felicia Torres"
+#swap_name = real_name.split(" ").reverse.join(" ")
+#swap_name = swap_name.downcase
+
+#Change vowels in name to next vowels
+def next_vowel(name)
+	vowels = ["a", "e", "i", "o", "u"]
```

Comment:
In general, constants are a great way to separate out the real-world data that doesn't change from variables that are being manipulated.